EXTENSIONS.md

```diff
@@ -0,0 +1,75 @@
+
+# Table of Contents
+- [About the core package](#ibm-generative-ai-core-package)
+- [What is an extension?](#what-is-an-ibm-generative-ai-extension)
+- [Types of extensions](#ibm-generative-ai-extension-types)
+- [Extensions migration](#extensions-migration)
+- [Design considerations](#design-considerations-to-build-extensions)
+
+# IBM Generative AI Core Package
+[IBM Generative AI open-source repository](https://github.com/IBM/ibm-generative-ai) lives under [IBM Github organization](https://github.com/IBM/). This repository is the official and unique location of IBM Gen AI core package source code.
+
+# What is an IBM Generative AI Extension?
+An extension is a software add-on that is installed on a program to enhance its capabilities. Gen AI extensions are meant to expand the capabilities of the Gen AI core package. For instance, imagine that you want to build a prompt pattern with data from a pandas dataframe. You could write your own code  to achive such functionaliy, or you could instead use the Gen AI Pandas extension. Using the extension will allow you to save development time and deliver an elegant solution that re-uses code when it is possible.
+
+# IBM Generative AI Extension types
+IBM Generative AI supports two types of open-source extensions: open-source official, and open-source third-party extensions.
+
+## Open-source "Gen AI official" extensions
+Extensions that are meant for public use from get-go should instead be developed as open-source extensions. Examples of open-source extensions that have been already released as open-source are LangChain, Pandas, and Hugging Face extensions.
+
+### Ownership and location
+Open-source extensions should be submitted directly to the [IBM Generative AI open-source repository](https://github.com/IBM/ibm-generative-ai) and should be developed following the [open-source Gen AI contribution guide](https://github.com/IBM/ibm-generative-ai/blob/main/DEVELOPMENT.md). Open-source official extensions are typically developed by the Gen AI team, or in collaboration with them. Providing maintenance to open-source official extensions is responsability of the Gen AI team.
+
+## Open-source "third-party" extensions
+We refer to all other extensions not implemented nor officially maintained by the Gen AI team as open-source third-party extensions.
+
+### Ownership and location
+Open-source third-party extensions can live anywhere, inside [IBM organization](https://github.com/IBM/), the wider Github universe, or other contributors' own Github spaces (e.g., other IBM teams, clients, partners). Maintaining these extensions would be the responsibility of their owners.
+
+# Extensions migration
+Extensions might be migrated from one type to another upon evaluation and approval of the Gen AI maintenance team. Below, we describe three common migration scenarios:
```

Comment:
I think there's only 2 scenario at the moment.
```suggestion
Extensions might be migrated from one type to another upon evaluation and approval of the Gen AI maintenance team. Below, we describe two common migration scenarios:
```

Reply:
right, good catch!